src/modules/job-manager/job-manager.c

```diff
@@ -107,6 +145,10 @@ int mod_main (flux_t *h, int argc, char **argv)
     }
     zhashx_set_destructor (ctx.active_jobs, job_destructor);
     zhashx_set_duplicator (ctx.active_jobs, job_duplicator);
+    if (!(ctx.note_requests = zlist_new ())) {
+        flux_log_error (h, "error creating note_requests list");
```

Comment:
You may need to manually set `errno = ENOMEM` here since `zlist_t` makes no guarantees about errno.